sonyapilib/device.py

```diff
@@ -359,12 +360,25 @@ def _parse_dmr(self, data):
             for service in device:
                 service_id = service.find(
                     f"{URN_UPNP_DEVICE}serviceId")
-                if "urn:upnp-org:serviceId:AVTransport" not in service_id.text:
-                    continue
-                transport_location = service.find(
-                    f"{URN_UPNP_DEVICE}controlURL").text
-                self.av_transport_url = f"{lirc_url.scheme}://{lirc_url.netloc.split(':')[0]}"\
-                    f":{self.dmr_port}{transport_location}"
+                service_id_text = service_id.text
+                # print(service_id_text)
```

Comment:
remove comment (all commented out print statements you added )

Reply:
Done 👍🏻